admin/gv_queue.php

```diff
@@ -162,7 +163,7 @@ function init()
   $gv_split = new splitPageResults($_GET['page'], MAX_DISPLAY_SEARCH_RESULTS, $gv_query_raw, $gv_query_numrows);
   $gv_list = $db->Execute($gv_query_raw);
   while (!$gv_list->EOF) {
-    if (((!$_GET['gid']) || (@$_GET['gid'] == $gv_list->fields['unique_id'])) && (!$gInfo)) {
+    if (((!isset($_GET['gid'])) || (@$_GET['gid'] == $gv_list->fields['unique_id'])) && (!isset($gInfo))) {

```

Comment:
Won't this get bypassed if `$_GET['gid'] == 0`, `$_GET['order']` is not provided even if `$gInfo` isn't yet set?

In other words shouldn't the first `isset` be `empty`?